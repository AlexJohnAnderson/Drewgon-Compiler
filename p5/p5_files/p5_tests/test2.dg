bool b;
int foo(int a){
b = true;
return b;
}

