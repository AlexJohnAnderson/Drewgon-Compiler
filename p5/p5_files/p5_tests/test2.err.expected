FATAL [4,8]-[4,9]: Bad return value
Type Analysis Failed
